src/main/java/hudson/scm/SubversionRepositoryStatus.java

```diff
@@ -146,9 +146,22 @@ public void doNotifyCommit(StaplerRequest req, StaplerResponse rsp) throws Servl
 
         rsp.setStatus(SC_OK);
     }
+
+    public static class SubversionRepoUUIDAndRootPath {
```

Comment:
Is this intended to be new public plugin API, or could this be `@Restricted(NoExternalUse.class)`?